examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.cpp

```diff
@@ -27,6 +29,16 @@
 
 using namespace ::chip;
 
+namespace {
+
+// Constants
+constexpr uint32_t kCommissionPrepareTimeMs = 500;
+constexpr uint16_t kMaxManaulCodeLength     = 11;
```

Comment:
I don't know exactly because I haven't analyzed all of this Spec, but does the Manual Pairing Code only use Short Manual Pairing in Fabric Sync? If you use the Manual Pairing Code with Vendor ID and Product ID support, it should be 21.